README.md

```diff
@@ -1 +1,94 @@
-# Villa-Iluminada
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My First Capstone
+
+This is my first professional capstone I created for Microverse.
+You can see it in this link
+https://ivaan1992.github.io/Villa-Iluminada/
+
+This project consists in making my own web page.
+
+For this project I thought about an event that ocurrs each December, in my state.
+
+I worked pretty harder.  
+
+
+This is the <a href=https://www.behance.net/gallery/29845175/CC-Global-Summit-2015>template</a> that I should use for making this project. Original design idea by Cindy Shin in Behance.
+
+<b><h3>MOBILE VERSION</h3></b> 📱
+
+To my capstone mobile version from this template I followed these steps: 
+
+1-. Made the header which includes the hamburguer icon menu and added some medias. 
+
+2-. Made the headline section which includes a description of the event, and added a picture.  
+
+3-. Made the program section which includes the some cards that show us everything there is in the event. 
+
+4. Finally made the speakers section which includes random people, but some of them are envolved in the event, but added a JavaScript function in the botton. 
+
+5-. After that, I made the about web page.  
+
+<b>This is the Mobile Version</b>
+
+![screenshot](https://user-images.githubusercontent.com/73128809/133906912-c9ba57aa-fc59-4d17-a610-4a34a6462b8e.png)
+
+<b><h3>DESKTOP VERSION</h3></b> 💻 🖥️
+To create Desktop Version I followed these steps: 
+
+1-. Opened another CSS file.
+
+2-. Copied and edited the original code.
+
+3-. Added media queries.
+
+4-. Added images, edited some texts, edited the form, and take off the last line. 
+
+<b>This is the Desktop Version</b>
+
+![screenshot](https://user-images.githubusercontent.com/73128809/133906960-3cadad1d-6d2f-4949-ba6c-8a79318f5032.png)
+
+
+
+## Built With ⚒️💻
+
+➡️HTML
+  
+➡️CSS
+
+➡️JAVASCRIPT
+
+➡️VISUAL STUDIO CODE
+
+
+## Live Demo
+
+
+[Live Demo Link](https://ivaan1992.github.io/Villa-Iluminada/)
+
+
+
+
+## Author
+
+👤 **Ivan Gaona**
+
+
+- GitHub: [@ivaan1992](https://github.com/ivaan1992)
+- Twitter: [@ivangaona92](https://twitter.com/ivangaona92)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ivan-linares-gaona/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] It would be nice that you fix the issues link for users who feel free and want to navigate from the README, what do you think? 🤔

Reply:
I just check the link and it works normally. The header is fixed and I don't know why this font looks like this now but is the same font of the design!! 
![Captura de pantalla de 2021-09-19 02-09-25](https://user-images.githubusercontent.com/73128809/133918842-2609bbd1-2735-4f46-a3d4-61fd566ef249.png)



.